File: jddata/app/src/main/java/com/example/jddata/util/ScreenUtils.java
package com.example.jddata.util;

import android.os.Environment;

import com.example.jddata.util.ExecUtils;

public class ScreenUtils {
    public static void scrrenShot() {
        String command = "screencap -p " + Environment.getExternalStorageDirectory().getAbsolutePath() + "/Pictures/" + System.currentTimeMillis() + ".png";
        ExecUtils.handleExecCommand(command);
    }
}

File: jddata/app/src/main/java/com/example/jddata/util/ScreenUtils.kt
package com.example.jddata.util

import android.os.Environment

import com.example.jddata.util.ExecUtils

class ScreenUtils {
    companion object {
        @JvmStatic fun scrrenShot() {
            val command = "screencap -p " + Environment.getExternalStorageDirectory().absolutePath + "/Pictures/" + System.currentTimeMillis() + ".png"
            ExecUtils.handleExecCommand(command)
        }
    }
}
